### ISSUE TITLE
Can't share photos

### ISSUE THREAD

**Issue by Linerly** [vector-im/element-android#4545]
### Steps to reproduce

1. Go to a DM
2. + → Gallery
3. Pick multiple photos (or just one)

#### What did you expect?
The photo(s) that I selected gets sent

#### What happened instead?
Element Android crashes

### Your phone model

Mito W1

### Operating system version

Android 9

### Application version and app store

1.3.8 [40103082] (G-b4480)

### Will you send logs?

Yes

**Comment by nicoduck** [comment]
This happens also on an Ulefone Armor 7 running Android 9.
Element installed via play Store, latest version (1.3.9).

**Comment by ouchadam** [comment]
https://github.com/matrix-org/element-android-rageshakes/issues/28358

from the logs it appears your devices graphics driver crashed, unfortunately there's not much we can do about this from the app side.

```
Fatal signal 6 (SIGABRT), code -6 (SI_TKILL) in tid 10624 (RenderThread), pid 10553 (im.vector.app)
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Native Crash TIME: 5761382
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'MITO/MITO_A36_W1/MITO_A36_W1:9/P00610/1564711811:user/release-keys'
Revision: '0'
ABI: 'arm64'
pid: 10553, tid: 10624, name: RenderThread  >>> im.vector.app <<<
signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
Abort message: 'glEGLImageTargetTexture2DOES error! GL_INVALID_OPERATION (0x502)'
```

are you able to select images from the file browser instead of the gallery? if so I would lean towards this being a gallery app issue~

**Comment by Linerly** [comment]
> are you able to select images from the file browser instead of the gallery?

Not really

**Comment by ouchadam** [comment]
to double check, do other apps crash when sharing images from the stock gallery?

**Comment by Linerly** [comment]
No, other apps like WhatsApp doesn't crash when I share photos from the stock gallery (which is Google Photos)

**Comment by Linerly** [comment]
Though Element Android doesn't crash on Samsung (Galaxy J1 ace) when I try to send a photo from Element Android (+ → Gallery → Gallery (default is file picker on Samsung))

**Comment by ouchadam** [comment]
for the flow that crashes on the `Mito W1`, when you say gallery app, do you mean the stock app or google photos? does google photos have the same crash?

when I try to use google photos from the `+` flow it only shows videos (I'll raise as a separate issue)

**Comment by Linerly** [comment]
The stock app is Google Photos, the gallery app I was using is [Simple Gallery Pro](https://f-droid.org/en/packages/com.simplemobiletools.gallery.pro/)

**Comment by Linerly** [comment]
…apparently I can send screenshots but not pictures taken by my phone
